sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -408,6 +408,10 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     });
   }
 
+  async doesExist(guid: string): Promise<number> {
+    return this.repo.count({ guid: guid });
```

Comment:
Please avoid using counting to check if a record exists. The count will force the query to process all the records in the table instead of stopping on the first match. Please consider changing it to search using a "where" like syntax.

Reply:
As per discussion we are going to stick with count in this scenario.